great_expectations/agent/models.py

```diff
@@ -43,6 +44,11 @@ class RunColumnDescriptiveMetricsEvent(EventBase):
     data_asset_name: str
 
 
+class DraftDatasourceConfig(EventBase):
+    type: Literal["test_datasource_config"] = "test_datasource_config"
```

Comment:
A bit confused by the naming. Should class be `TestDraftDatasourceConfig` or `TestDatasourceConfig`? To stick with convention, should we prepend `Run` and append `Event` to the class name? 

Also, all the other event type fields have `.received` at the end so far - should this also follow that?

Reply:
Doesn't this Event need to be connected to Action in the Eventhandler?

Reply:
yes, thanks!

Reply:
thanks for the questions - `TestDraftDatasourceConfig` caused issues with pytest, since it tries to pick it up as a test. `.received` should be removed in new actions - it's not doing anything, and we don't have any plans to use other suffixes. The `Run` prefix isn't a broad convention, it's a part of the action name, i.e. "Run Data Assistant". In this case, we're not running anything, so isn't necessary.

Reply:
I think we could get around this if we wanted to prefix with `Test`: https://stackoverflow.com/a/24635769

Reply:
thanks @cdkini - not sure it's worth the effort to try and work around pytest convention, the rename is clear enough 👍